wasi/wasi.go

```diff
@@ -101,65 +144,99 @@ type api struct {
 
 func (w *api) register(store *wasm.Store) (err error) {
```

Comment:
if you don't mind, how about renaming these old `(w *api)` variable to `a` as well? They are remnant of `(w *wasiEnvironment)`

Reply:
sure thing